src/commands/remove-demo.ts

```diff
@@ -15,14 +31,19 @@ module.exports = {
     p()
     p(`Removing demo code from '${TARGET_DIR}'`)
 
-    const getAllFilePaths = createGetAllFilePaths(filesystem)
-    const paths = getAllFilePaths(TARGET_DIR).filter(
-      (path) => path.includes("node_modules") === false,
-    )
+    const filePaths = filesystem
+      .cwd(TARGET_DIR)
+      .find({
+        matching: MATCHING_GLOBS,
+        recursive: true,
+        files: true,
+        directories: false,
+      })
+      .map((path) => pathlib.join(TARGET_DIR, path))
```

Comment:
I removed the need for the `path.ts` helper since we can achieve what we need through this. this should find all files that exclude the paths defined above using the globs array